features/invitelist/impl/src/main/kotlin/io/element/android/features/invitelist/impl/InviteListPresenter.kt

```diff
@@ -51,10 +53,19 @@ class InviteListPresenter @Inject constructor(
             .roomSummaries()
             .collectAsState()
 
-        val seenInvites = remember { store.seenRoomIds }
+        val seenInvites = remember { mutableStateOf<Set<RoomId>>(emptySet()) }
+
+    LaunchedEffect(Unit) {
+            seenInvites.value = store.seenRoomIds().first()
+        }
 
         LaunchedEffect(invites) {
-            store.seenRoomIds = invites.map { it.identifier() }.toSet()
+            store.markAsSeen(
```

Comment:
It will mark as seen all invites, if the list is long and invites are not all visible, it's maybe not the best? Maybe you should track the visible items in the UI?

Reply:
Agree that might be nice eventually, but the current behaviour is what was designed/specced (and has already been implemented on iOS): 

![image](https://user-images.githubusercontent.com/189133/235629997-515472c3-9a4b-45e8-8cb8-d8fd12a79043.png)